seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-3/src/test/resources/jdbc_hive_source.conf

```diff
@@ -0,0 +1,46 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+######
+###### This config file is a demonstration of streaming processing in seatunnel config
+######
+
+env {
+  # You can set engine configuration here
+  execution.parallelism = 1
+  job.mode = "BATCH"
+  checkpoint.interval = 5000
+  #execution.checkpoint.data-uri = "hdfs://localhost:9000/checkpoint"
+}
+
+source {
+  # This is a example source plugin **only for test and demonstrate the feature source plugin**
+    Jdbc {
+        url = "jdbc:hive2://localhost:10000/default"
+        user = "root"
+        driver = "org.apache.hive.jdbc.HiveDriver"
```

Comment:
We currently do not have hive E2e tests, can your E2e directly cover jdbc tests and hive thrift mode read and write, metastore
https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/source/Hive.md,https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/sink/Hive.md

Reply:
> https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/source/Hive.md,https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/sink/Hive.md

`jdbc_hive` and `hive` are difference connector, It would be better to separate the e2e of the two connectors. We are missing the e2e for the hive connector. 

@NickYoungPeng  If you are willing, you can provide a separate PR to add the e2e for the hive connector.